### PR TITLE
price-info does not depend on util

### PR DESCRIPTION
**Pull request by debris**
tiny pr, 

- first commit removes dependency on util and moves price-info.rs -> lib.rs
- second commit makes a code a bit more idiomatic

### CODE REVIEW

File: util/fetch/src/client.rs [hunk 1]
```diff
@@ -297,7 +297,7 @@ impl Response {
 
 	/// Returns `true` if response status code is successful.
 	pub fn is_success(&self) -> bool {
-		self.status() == reqwest::StatusCode::Ok
+		self.status().is_success()
```
Comment (tomusdrw) [inline util/fetch/src/client.rs]: Why was this changed? AFAIR `is_sucess()` returns true for any 2xx status code and I think we should be more strict in that case.

Comment (debris) [inline util/fetch/src/client.rs]: I see, will revert it

**Comment by debris** [comment]
CI fail is unrelated
